README.md

```diff
@@ -25,6 +26,27 @@ Artifacts will be written to `build/libs`. Use `appmap.jar` as your agent.
 ```
 $ ./gradlew build
 ```
+# Configuration
+When you run your program, the `javaagent` reads configuration settings from `appmap.yml`. Here's a sample configuration file for a typical Java project:
```

Comment:
Elsewhere in the README, we use "agent" to refer to the agent. So, this should be "the agent reads", rather than "the \`javaagent\` reads".